ipfilter.go

```diff
@@ -175,7 +176,7 @@ func (f *IPFilter) initDB() error {
 	defer resp.Body.Close()
 	//store on disk as db loads
 	r := io.TeeReader(resp.Body, file)
-	err = f.readerDB(DBPublicURL, r)
+	err = f.readerDB(f.opts.IPDBPath, r)
```

Comment:
This line should be `f.opts.IPDBFetchURL` I think...

Reply:
In `readerDB(filename string, r Reader)`, the first argument just as a FileName for gzReader, not a URL. The Correct Suffix(gz or not gz) of IPDB, decision is developer options.